incubator/fairwinds-metrics/templates/controller-deployment.yaml

```diff
@@ -53,6 +53,14 @@ spec:
             - controller
           args:
             - -v{{ .Values.controller.logVerbosity }}
+          {{- if .Values.listener.enabled }}
+            - --event-listener=true
+            - --events-port={{ .Values.listener.port }}
+            - --events-username={{ .Values.listener.username }}
+            - --events-password={{ .Values.listener.password | quote }}
```

Comment:
Would it be possible to pass this as a secretRef and pull from env instead? I'm not a fan of passing secrets to helm in plain text.

Reply:
sure, will take a few updates to the app

Reply:
i added a secret & moved the username/password there.  This requires https://github.com/FairwindsOps/custom-metrics/pull/19 to be merged.  I also updated the image tag to prep.